tests/components/sensibo/response.py

```diff
@@ -0,0 +1,400 @@
+"""Test api response for the Sensibo integration."""
+from __future__ import annotations
+
+from pysensibo.model import MotionSensor, SensiboData, SensiboDevice
+
+DATA_FROM_API = SensiboData(
+    raw={
```

Comment:
We normally put data like this in a json file in a fixtures directory for the tests of the integration. Then load the data in a pytest fixture and return it as needed.

Reply:
Alright. I do that in a new PR before I add the remaining tests.

Reply:
Rethinking: This is a dataclass coming from the upstream package where "raw" is just a minor subset to have for diagnostics. 
Not sure if you mean to get the whole dataclass into a json file (seems not ideal) or just the section of "raw"?

Reply:
How you compose the fixture data into files is up to you and how the library response is built.